pkg/resources/materialized_view_acceptance_test.go

```diff
@@ -0,0 +1,73 @@
+package resources_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAcc_MaterializedView(t *testing.T) {
+	accName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+
+	resource.ParallelTest(t, resource.TestCase{
+		Providers: providers(),
+		Steps: []resource.TestStep{
+			{
+				Config: materializedViewConfig(accName, []string{"c1", "c2"}, "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_materialized_view.test", "name", accName),
+					resource.TestCheckResourceAttr("snowflake_materialized_view.test", "database", accName),
+					resource.TestCheckResourceAttr("snowflake_materialized_view.test", "schema", accName),
+					resource.TestCheckResourceAttr("snowflake_materialized_view.test", "comment", "Terraform test resource"),
+					resource.TestCheckResourceAttr("snowflake_materialized_view.test", "cluster_by", "[\"c1\",\"c2\"]"),
+					checkBool("snowflake_materialized_view.test", "is_secure", true),
+				),
+			},
+		},
+	})
+}
+
+func TestAcc_MaterializedView2(t *testing.T) {
+	accName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
```

Comment:
```suggestion
	accName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
```

Snowflake is case sensitive